application/models/Permission.php

```diff
@@ -65,6 +65,18 @@ public static function model($class = __CLASS__)
      * @return array
      */
     public static function getSurveyBasePermissions()
+    {
+        return self::getEntityBasePermissions('Survey');
+    }
+
+    /**
+     * Return Permission for an object
+     * @param string $sEntityName muts be an existing object child of LSActiveRecord
+     * @access public
```

Comment:
@access is not needed

Reply:
Right : copying fro surveyBasePermission …